### PR TITLE
docs(kmesh) expiration and revocation of cp token

### PR DESCRIPTION
**Pull request by jakubdyszkiewicz**
### Summary

Add information about expiration and revocation

### CODE REVIEW

File: app/mesh/1.4.x/features/kds-auth.md
Comment (mheap) [inline app/mesh/1.4.x/features/kds-auth.md:200]: If a signing key is comprimised, would that automatically invalidate all tokens once rotated, or do you also have to add all of those tokens to the revocation list?

Comment (jakubdyszkiewicz) [reply app/mesh/1.4.x/features/kds-auth.md:200]: Once you remove the compromised signing key, all tokens signed with this signing key are no longer valid so you don't have to manually add them to the revocation list.

See `Remove the old signing key` - `All new connections to the global control plane now require tokens signed with the new signing key`

**Review by Bradamant3 — APPROVED** [review]
LGTM, will edit lightly after merge

**Pull request merged**
